build-artifacts.py

```diff
@@ -28,9 +29,9 @@ def __init__(self):
 
         # Setup logging level
         if arguments.debug:
-            logging.basicConfig(level=logging.DEBUG)
+            logging.basicConfig(level=logging.DEBUG, format='%(message)s')
         else:
-            logging.basicConfig(level=logging.INFO)
+            logging.basicConfig(level=logging.INFO, format='%(message)s')
```

Comment:
I think so. If the `format` not provided, I got the ugly prefix as shown:

```
WARNING:root:[Warning] These parking lot OBJECTID's were ignored because they don't have a valid Prop_ID or ZoneGroup
```

And here is the example if `format='%(message)s'` is added:

```
[Warning] These parking lot OBJECTID's were ignored because they don't have a valid Prop_ID or ZoneGroup:
```

Reply:
Nevermind, I decided to remove `[Warning]` so I'll also remove the format here.